repositories.yaml

```diff
@@ -964,6 +964,27 @@ applications:
         additional_dependencies:
           - glean-core
 
+  - app_name: mozillavpn_cirrus
+    canonical_app_name: Mozilla VPN Cirrus Sidecar
+    app_description: |
+      The sidecar Cirrus container for the Mozilla VPN server
+    url: https://github.com/mozilla/experimenter
+    notification_emails:
+      - brizental@mozilla.com
+    branch: master
```

Comment:
The default branch is `main` and there is no `master` branch, but GitHub seems to redirect to the default branch if you try to access a nonexistent branch (at least on the website), so maybe this will still work?